src/main/java/com/example/music/backend/user/converter/UserDtoConverter.java

```diff
@@ -0,0 +1,27 @@
+package com.example.music.backend.user.converter;
+
+import com.example.music.backend.user.domain.User;
+import com.example.music.backend.user.dto.UserDto;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserDtoConverter {
+    private final PasswordEncoder encoder;
+
+    public UserDtoConverter(PasswordEncoder encoder) {
+        this.encoder = encoder;
+    }
+
+    public User toEntity(UserDto dto) {
```

Comment:
do kind of smell a bit. Doesn't looks like SRP

Reply:
now the password is encoded in the UserController